lib/events.js

```diff
@@ -52,8 +59,10 @@ Object.defineProperty(EventEmitter, 'defaultMaxListeners', {
     console;
     // check whether the input is a positive number (whose value is zero or
     // greater and not a NaN).
-    if (typeof arg !== 'number' || arg < 0 || arg !== arg)
-      throw new TypeError('"defaultMaxListeners" must be a positive number');
+    if (typeof arg !== 'number' || arg < 0 || arg !== arg) {
+      const errors = lazyErrors();
+      throw new errors.TypeError('ERR_EVENTS_MAX_LISTENERS_TYPE');
```

Comment:
I do not really see the point for this new error type. I it extremely specific and the type name is somewhat confusing for me. I would expect it to describe having e.g. to many listeners. Using a more generic one that is similar to the old error message seems more appropriate to me.

Reply:
The challenge is that none of the existing error codes quite fit this pattern (e.g. it's a setter and not an argument or an option).